Meta/build-root-filesystem.sh

```diff
@@ -163,6 +163,7 @@ ln -s ProfileViewer mnt/bin/pv
 ln -s WebServer mnt/bin/ws
 ln -s Solitaire mnt/bin/sl
 ln -s WebView mnt/bin/wv
+ln -s test mnt/bin/\[
```

Comment:
This isn't really a shortcut.

Reply:
It was suggested by @bugaevc, so...what say you, @bugaevc?

Reply:
`[` is a shell builtin, not a special character like `(`, `)`, `;`, `&`, `\`, `|`, etc.

You don't need to escape builtins: `echo alias` works fine, you don't need to use `echo \alias`

Reply:
But `[` is not a shell builtin in our case. And I do think `[` can be considered a shortcut for `test` (although an unusual one, because it requires the matching `]`).

Reply:
So what's the verdict? should that `ln` stay where it is?
I have no particular preference, so whatever y'all decide on is fine.

Reply:
I think that `[` shouldn't be considered a shortcut for `test` since it has a different behaviour.

Reply:
Although even Dr. POSIX says it's fine, and that `[` is a "second form" of `test`.
> https://pubs.opengroup.org/onlinepubs/9699919799/utilities/test.html
> In the second form of the utility, where the utility name used is [ rather than test, [...]. The test and [ utilities may be implemented as a single linked utility

Reply:
Of course, that's how it works everywhere. It wasn't me who came up with symlinking it this way :smile:

Reply:
I'm not against symlinking!
I just think that `ln -s test mnt/bin/[` fits better in the `installing base system... ` section rather than in the `installing shortcuts... ` section.

https://github.com/SerenityOS/serenity/pull/2618#issuecomment-649044754